open-company/open-company-web
# Compose button

Card: https://trello.com/c/E809Ds0X

To test:
- open AP
- plus button in top right is grey?
- scroll down to partially cover QP
- plus button turns green?
- click it
- close Cmail
- switch to a section
- expand QP
- current section is the default one?
- close QP
- add a new section
- expand QP
- new section is selected?
- switch to another section
- check it's selected in QP
- delete the section
- check the deleted section is no longer selected in QP
- check the deleted section is no longer in the sections list of QP
- type something in QP
- click the + button in top right
- still editing the same post but in fullscreen?
- play a bit with QP and FP and sections switching
